packages/components/src/EditableText/EditableText.component.js

```diff
@@ -11,17 +11,18 @@ import getDefaultT from '../translate';
 
 import I18N_DOMAIN_COMPONENTS from '../constants';
 
-export function PlainTextTitle({ onEdit, disabled, text, inProgress, feature, t }) {
+export function PlainTextTitle({ componentClass, onEdit, disabled, text, inProgress, feature, t }) {
 	const isDisabled = disabled || inProgress;
+	const ComponentClass = componentClass;
 	return (
 		<div className={theme['tc-editable-text-title']}>
 			<TooltipTrigger label={text} tooltipPlacement="bottom">
-				<span
+				<ComponentClass
```

Comment:
you can t write ```<componentClass``` ?

Reply:
No, react consider it as a native markup